pyproject.toml

```diff
@@ -33,9 +34,12 @@ pytest = ">=7.1.3"
 pytest-cov = ">=4.0.0"
 tox = ">=3.26.0"
 openpyxl = ">=3.0.5"
+pyarrow = ">=9.0.0"
```

Comment:
I would rather install for dev with `poetry install -E all`. Having pyarrow as both a dependency and dev dependency runs the risk of them getting out of sync.

I see I did the same with openpyxl so this is a note to myself. I will update both after this merges nothing you need to do :smile: